### PR TITLE
Add hints to the rerendering error message

### PR DESCRIPTION
**Pull request by chrisburr**
I think this might be useful for maintainers, thoughts?

(Follow-up to https://github.com/conda-forge/geckodriver-feedstock/pull/17#issuecomment-763293869)

### CODE REVIEW

File: webservices_dispatch_action/rerendering.py [hunk 1]
```diff
@@ -80,11 +80,22 @@ def comment_and_push_per_changed(
...
+                "blob/master/recipe/conda_build_config.yaml"
+            )
             message = """\
 Hi! This is the friendly automated conda-forge-webservice.
 I tried to rerender for you but ran into some issues, please ping conda-forge/core
 for further assistance. You can also try [re-rendering locally]({}).
-""".format(doc_url)
+
+More information on the issue may be available in the actions tab of the feedstock
+under a "rerender" workflow. The following suggestions might explain the problem:
```
Comment (bollwyvl) [inline webservices_dispatch_action/rerendering.py:115]: I don't really know what a rerender workflow is... one of the bot magic spells?

File: webservices_dispatch_action/rerendering.py [hunk 2]
```diff
@@ -80,11 +80,22 @@ def comment_and_push_per_changed(
...
+
+* Is the `recipe/meta.yaml` file valid?
+* If there is a `recipe/conda-build-config.yaml` file in the feedstock make sure
+  that it is compatible with the current [global pinnnings]({}).
```
Comment (bollwyvl) [inline webservices_dispatch_action/rerendering.py:119]: the link to the pinnings is really good. While I might have figured out that `cudnn` was the culprit, I wouldn't have necessarily been able to determine I should use `undefined`... i guess be process of elimination on the selectors?

File: webservices_dispatch_action/rerendering.py (hunk 1 above)
Comment (chrisburr) [reply webservices_dispatch_action/rerendering.py:115]: If you look on the "actions" tab of the feedstock: https://github.com/conda-forge/geckodriver-feedstock/actions

"workflow" is then the name for the pipeline/"type of CI job" in GitHub actions lingo. Any ideas for how to phrase it more clearly?

File: webservices_dispatch_action/rerendering.py (hunk 2 above)
Comment (chrisburr) [reply webservices_dispatch_action/rerendering.py:119]: For the specific case in https://github.com/conda-forge/geckodriver-feedstock/pull/17, the value(s) of `cudnn` are also in the pinnings (I think the other values aren't needed as CUDA isn't actually being used but I'm getting out of my depth at this point).

https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/d8507e01df35678a41ad432e9dd3278f73a689e0/recipe/conda_build_config.yaml#L47-L52

**Review by beckermr — CHANGES_REQUESTED** [review]
I bet if we are clever we can backout the direct link to the github action the made the comment. I should have added that when I wrote thus. :/

File: webservices_dispatch_action/rerendering.py (hunk 1 above)
Comment (bollwyvl) [reply webservices_dispatch_action/rerendering.py:115]: Ah, ok... so it's not a workflow i, the maintainer, requested, but a bot one.

The cadillac would be to generate a link to the workflow in question, as even the failed workflow comes up green... but it should be the last one. Failing that, is the current feedstock in scope for this message?

```
https://github.com/conda-forge/{{ feedstock }}/actions?query=workflow%3A.github%2Fworkflows%2Fwebservices.yml
```

**Comment by bollwyvl** [comment]
>  backout the direct link

and, once you get there, you could [throw a warning annotation](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-error-message) in for good measure (otherwise is green)...

**Comment by bollwyvl** [comment]
Some related stuff:
https://github.com/utgwkk/pytest-github-actions-annotate-failures/blob/master/pytest_github_actions_annotate_failures/plugin.py#L61

**Comment by bollwyvl** [comment]
Maybe the link is:

```py 
this_run_url = "{GITHUB_SERVER_URL}/{GITHUB_REPOSITORY}/actions/runs/{GITHUB_RUN_ID}".format(**os.environ)
```

File: webservices_dispatch_action/rerendering.py [hunk 3]
```diff
@@ -80,11 +80,22 @@ def comment_and_push_per_changed(
...
+""".format(doc_url, global_pinning_url)
```
Comment (beckermr) [inline webservices_dispatch_action/rerendering.py:122]:
```suggestion
* Is the fork used for this PR on an organization or user GitHub account? Automated rerendering via the 
  webservices admin bot only works for user GitHub accounts.
""".format(doc_url, global_pinning_url)
```

File: webservices_dispatch_action/rerendering.py (hunk 1 above)
Comment (beckermr) [reply webservices_dispatch_action/rerendering.py:115]: the workflow link should be reconstructable given the variables github puts in the environment.

**Comment by beckermr** [comment]
See the circle build url here

```
          export CIRCLE_BUILD_URL="https://github.com/regro/autotick-bot/actions/runs/${RUN_ID}"
          export CIRCLE_BUILD_NUM="actually-actions-${RUN_ID}"
          pushd cf-graph
          conda activate run_env
          conda-forge-tick --run 3 ${BOT_CLI_ARGS}
          popd
        env:
          USERNAME: regro-cf-autotick-bot
          PASSWORD: ${{ secrets.AUTOTICK_BOT_TOKEN }}
          RUN_ID: ${{ github.run_id }}
```

Looks like we only need the feedstock name and the run id.

**Comment by beckermr** [comment]
oh woops missed the note above

🤦

**Pull request closed without merging**